### ISSUE TITLE
4-finger swipe has a weird acceleration while swiping

### ISSUE THREAD

**Issue by intitni** [intitni/AbnormalMouseApp#1]

**Comment by agnesnutter** [comment]
I'm not sure about weird acceleration, but something I'm specifically looking for is to be able to swipe between spaces with analog motion (like four fingers on the trackpad) rather than activating a "move to the next space" single action. The latter I can achieve any number of ways, and seems to be what Abnormal Mouse is doing... is it possible to simulate four fingers moving like you do for the two-finger gestures?

**Comment by intitni** [comment]
Hi @agnesnutter 

It's already there in Abnormal Mouse. You can hold the activator for the 4-finger gesture feature and move your mouse around, the spaces should move accordingly. 

It doesn't feel as nice as a trackpad now, but I find it acceptable. 

But if anyone thinks an improvement is needed, I will take a look into it when I have time.

**Comment by intitni** [comment]
@agnesnutter Yes that's what I mean. It works fine on my side though. Can you tell me which macOS version you are using, and what the activator set to this feature?

**Comment by intitni** [comment]
@agnesnutter also which version of Abnormal Mouse are you using? I remember there is an old version that has a bug that activators will be released automatically. But that's pretty old.
